content/en/v3/admin/platforms/google/svc_acct.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configure Google Service Account
+linktitle: GCP Service Account
+type: docs
+weight: 100
+aliases:
+---
+
+This doc will demonstrate how to set up a Google service account that can be used by Terraform to execute [Jenkins X GKE Module](https://github.com/jenkins-x/terraform-google-jx#jenkins-x-gke-module)
+
+> ⚠️  This doc has been designed to assist in performing the demonstration through copying and pasting each block of code into a shell terminal.<br>
+>
+> To execute the commands listed in your local bash shell will require the [Google gcloud tool and Cloud SDK](https://cloud.google.com/sdk/gcloud/#the_and) and the [JQ command-line JSON processor](https://stedolan.github.io/jq/).<br>
+>
+> To execute the commands listed in [Google Cloud Shell](https://cloud.google.com/shell/) does not require any additiobnal software installed.
+
+### Service account privileges
+In order to build your GKE environment with Terraform using a service account, the following are the service accounts minimal role requirements:
+* roles/container.admin
+* roles/editor
+* roles/iam.serviceAccountAdmin
+* roles/iam.serviceAccountKeyAdmin
+* roles/resourcemanager.projectIamAdmin
+* roles/storage.admin
+
+If the service account needs to access a separate project to manage an apex domain then an additional role setting is required for the separate project.
+* roles/dns.admin
+
+### Create service account and assign roles
+The first step is to create a new service account (APP_NAME) and to assign tbe roles.
```

Comment:
typo: `assign tbe roles.`

Reply:
I use 'assign roles' in the ### line because its a summary statement and not a sentence. I use 'assign the roles' in the following line because it's a sentence.

Reply:
I meant the typo is in `the` being spelt as `tbe`.

Reply:
I need my glasses upgraded... sorry